pkg/apis/resource/v1alpha1/pipeline_resource_types.go

```diff
@@ -26,6 +26,20 @@ import (
 // Deprecated: Unused, preserved only for backwards compatibility
 type PipelineResourceType = string
 
+const (
```

Comment:
do we need to add this in v1alpha1? I cannot find any references in this pr

Reply:
Good question. This is where it used to be. When we converted back, this file was partially restored: https://github.com/tektoncd/pipeline/pull/6436/files.